src/main/scala/mesosphere/marathon/api/v2/PodsResource.scala

```diff
@@ -83,8 +84,11 @@ class PodsResource @Inject() (
     * @return HTTP OK if pods are supported
     */
   @HEAD
-  def capability(@Context req: HttpServletRequest): Response = authenticated(req) { _ =>
-    ok()
+  def capability(@Context req: HttpServletRequest, @Suspended asyncResponse: AsyncResponse): Unit = sendResponse(asyncResponse) {
+    async {
+      implicit val identity = await(authenticatedAsync(req))
```

Comment:
do you need the implicit val here? couldn't it be just a statement?

Reply:
The `implicit` is not required. This just came from copying and pasting.